extensions/protobuf/src/main/java/io/deephaven/protobuf/ToChunkTypeTransform.java

```diff
@@ -1,13 +1,9 @@
 //
 // Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
 //
-package io.deephaven.kafka;
+package io.deephaven.protobuf;
```

Comment:
This is not really an appropriate package for it, at least publicly (I also claim that `io.deephaven.kafka` is not appropriate, but in that case it was internal only).

That said, I'm _okay_ putting it into extensions-protobuf, but using a more appropriate package; `io.deephaven.function`; this sort of implies that we can continue to refactor it later to a more appropriate place if necessary.

Reply:
Done